docs/reference/migrations.md

```diff
@@ -24,12 +24,18 @@ You can always rollback some migrations specifing what version you would like to
 
 _Example_
 
-```
+```shell
```

Comment:
`sh` isn't supported out-of-the-box so we've used `bash` for other shell code blocks (not sure about `shell`).

Reply:
I just tried with `bash`, `shell`, `sh`, `console` and they all look the same (in docusaurus).
I don't think it exist an "official" list of language codes, the best I've found is this one, and according with this we can use `shell` (but also `sh`?). https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md

Reply:
They will look the same if the code block doesn't contain any keywords that have a different colour. The Docusaurus supported languages are here in the docs: https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages

And here in code: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js

`bash` is the only supported shell language out-of-the-box. Others can be added if needed, but I don't think that's necessary in this case — I would use `bash` as we have for other shell code blocks ([example](https://oss.platformatic.dev/docs/next/getting-started/quick-start-guide)).

On a related note, I'll look at documenting in the Documentation Style Guide what languages are supported for code blocks.

Reply:
OK, changing to `bash`, tnx for spotting this